draft-ietf-rats-corim.md

```diff
@@ -1803,37 +1803,88 @@ The selected tags are mapped to an internal representation, making them suitable
+
+* The issuer of the Endorsement conceptual message is copied to the `ev` `addition` ECT `authority` field.
+
+* If the Endorsement conceptual message has a profile, the profile is copied to the `ev` `addition` ECT `profile` field.
 
 #### Endorsement Triples Transformations {#sec-end-trans}
 
 Endorsement Triple Transformation :
 
-{:ett-enum: counter="bar" style="format Step %d."}
+{:ett-enum: counter="ett" style="format Step %d."}
 
 {: ett-enum}
 * An available `ev` entry ({{sec-ir-end-val}}) is allocated.
-The Endorsement Triple ({{sec-comid-triple-endval}}) `environment-map` is copied to the `environment-map` for both the `ev` `condition` and `ev` `addition` ECTs.
 
-* For each `measurement-map` entry in the measurements list, the i<sup>th</sup> `measurement-map` entry is copied to the i<sup>th</sup> entry in the `addition` ECT `elements` list.
+* The ECT `cmtype` of the `ev` entry's `addition` ECT is set to `endorsements`.
 
-* The issuer of the Endorsement conceptual message is copied to the `ev` `addition` ECT authority field.
+* The Endorsement Triple ({{sec-comid-triple-endval}}) `environment-map` is copied to the `environment-map` for both the `ev` `condition` and `ev` `addition` ECTs.
 
-* If the Endorsement conceptual message has a profile, the profile is copied to the `ev` `addition` ECT profile field.
+* For each `measurement-map` entry in the measurements list, the i<sup>th</sup> `measurement-map` entry is copied to the i<sup>th</sup> entry in the `addition` ECT `element-map` of the `element-list`.
+
+* The issuer of the Endorsement conceptual message is copied to the `ev` `addition` ECT `authority` field.
+
+* If the Endorsement conceptual message has a profile, the profile is copied to the `ev` `addition` ECT `profile` field.
 
 Conditional Endorsement Triple Transformation :
 
-> [^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/285
+{:cett-enum: counter="cett" style="format Step %d."}
+
+{: cett-enum}
+* An available `ev` entry ({{sec-ir-end-val}}) is allocated.
+
+* The ECT `cmtype` of the `ev` entry's `addition` ECT is set to `endorsements`.
+
+* For each entry in the Conditional Endorsement Triple ({{sec-comid-triple-cond-endors}}) `conditions` array, the `stateful-environment-record` is copied to a `condition` ECT in the `ev` entry.
+The `environment-map` of the `stateful-environment-record` is copied to the ECT `environment` field.
+
+* For each `measurement-map` entry in the measurements list of the `stateful-environment-record`, the i<sup>th</sup> `measurement-map` entry is copied to the i<sup>th</sup> entry in the `condition` ECT `element-map` of the `element-list`.
+
+* For each entry in the Conditional Endorsement Triple `endorsements` array, the `endorsed-triple-record` is copied to an `addition` ECT in the `ev` entry.
+The `environment-map` of the `endorsed-triple-record` is copied to the ECT `environment` field.
```

Comment:
i think the environment map of the `endorsed-triple-record` is the environment to which the Endorsement applies.

It should reside, where the addition ECT, i.e. place where the stuff gets added as Endorsement exist

Reply:
I'm not seeing where the issue is. The text seems fine to me since "the ECT" refers to the addition ECT that corresponds to an entry in the conditional endorsement triple's `endorsements` array.